examples/playground/src/App.tsx

```diff
@@ -51,10 +52,10 @@ function App() {
     },
     'ui:submitButtonOptions': {
       // working in the next release
-      norender: true,
+      norender: false,
       props: {
         className: 'btn btn-info',
-        disabled: true,
+        disabled: false,
       },
       submitText: 'Save',
     },
```

Comment:
As the next release is now available, what do you think to update @rjsf in this PR (in a dedicated commit)?